localstack-core/localstack/utils/analytics/usage.py

```diff
@@ -50,15 +50,47 @@ def aggregate(self) -> dict:
 
 class UsageCounter:
     """
-    Use this counter to count numeric values and perform aggregations
+    Use this counter to count numeric values
 
-    Available aggregations: min, max, sum, mean, median
+    Example:
+        my__counter = UsageCounter("lambda:somefeature")
+        my_counter.increment()
+        my_counter.increment()
+        my_counter.aggregate()  # returns {"count": 2}
+    """
+
+    state: int
+    namespace: str
+
+    def __init__(self, namespace: str):
```

Comment:
in order to not introduce a breaking change, we could still add `aggregations` in the signature of the class, even if it does nothing, in order to not break newly introduced usage in upstream dependencies

Reply:
Honestly probably not that important. Everyone is adapting the counters right now anyway, I'm fine with leaving it out here